examples/exit-code/src/lib.rs

```diff
@@ -4,6 +4,9 @@ use gstd::{msg, prelude::ToString, ActorId};
 
 static mut HOST: ActorId = ActorId::new([0u8; 32]);
 
+#[no_mangle]
+pub unsafe extern "C" fn handle() {}
```

Comment:
need to place comment here.
it was containing the only `handle_reply` previously, what is just for testing and isn't valid at all, cause we can't receive reply on message if we never send it

in tests we send message with source = id of this demo, to check `msg::exit_code()` functionality